### PR TITLE
chore: improve message send logging

### PR DESCRIPTION
**Pull request by tmspzz**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

There are no log traces for the message send flow for Beta builds

### Solutions

Trace message send flow with appropriate obfuscation for privacy in Beta builds

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by borichellow — APPROVED** [review]
Looks good to me

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/ScheduleNewAssetMessageUseCase.kt [hunk 1]
```diff
@@ -204,9 +204,7 @@ internal class ScheduleNewAssetMessageUseCaseImpl(
         message: Message,
         conversationId: ConversationId
     ): Either<CoreFailure, Unit> =
-        messageSender.sendPendingMessage(conversationId, message.id).onFailure {
-            kaliumLogger.e("There was an error when trying to send the asset on the conversation")
-        }
+        messageSender.sendPendingMessage(conversationId, message.id)
```
Comment (gongracr) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/ScheduleNewAssetMessageUseCase.kt:207]: Why are we trimming these error messages? I thought we wanted to be more verbose when it comes to logging errors?

Comment (gongracr) [reply logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/ScheduleNewAssetMessageUseCase.kt:207]: Nevermind, I just saw there was some generic error logging added in the underlying layers of the `MessageSender`

**Review by gongracr — APPROVED** [review]
LGTM 🗒️ 🖊️

**Pull request merged**